packages/@glimmerx/core/src/owner.ts

```diff
@@ -9,24 +9,37 @@ export interface FactoryIdentifier {
 
 const SERVICES = Symbol('Services');
 
+function isConstructor(func) {
+  return (
+    (func && typeof func === 'function' && func.prototype && func.prototype.constructor) === func
+  );
+}
+
 export default class Owner {
   private [SERVICES]: Dict<unknown>;
```

Comment:
Maybe just call this `SERVICES_CACHE` as its responsible for holding the cached referenced to the services.